documentation/asciidoc/topics/proc_creating_grafana_datasources.adoc

```diff
@@ -0,0 +1,64 @@
+[id='creating-grafana-dashboards_{context}']
+= Creating Grafana Data Sources
+Create a `GrafanaDatasource` CR so you can visualize {brandname} metrics in Grafana dashboards.
+
+.Prerequisites
+
+* Create a Prometheus `ServiceMonitor` CR for your {brandname} cluster.
+* Install the Grafana operator and create a Grafana deployment.
+
+.Procedure
+
+. Create a service account for the Grafana data source.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/prometheus_service_account.yaml[]
+----
++
+.. Apply the service account.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ {oc_apply_cr} service-account.yaml
+----
++
+.. Assign the `cluster-monitoring-view` role to the service account.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc adm policy add-cluster-role-to-user cluster-monitoring-view -z monitoring-account
+----
```

Comment:
I don't know if these stages are always required. Furthermore, these steps are different for the `kubectl` client. @rigazilla any thoughts?

Reply:
Thanks @ryanemerson I wasn't sure about this either and was digging into the `kubectl` commands but will see. Will remove if not required.

Reply:
This step is OCP only: an account with `cluster-monitoring-view` role is needed by the `GrafanaDataSource` to read data from the embedded Prometheus system.
If the user already has a service-account he can just use its token in the next step.